test/test_integration.py

```diff
@@ -54,6 +54,53 @@
     'expect_out': ['a,b,c', '1,2,3', '4,5,6']
 }
 
+comma_delimeter_csv = {
+    'name': 'comma_delimeter_csv',
+    'raw_data': ['a,b,c',
+                 '1,2,3',
+                 '4,5,6'],
+    'script': {"name": "comma_delimeter_csv",
+               "resources": [
+                   {"dialect": {
+                       "delimiter": ",",
+                       "do_not_bulk_insert": "True"
+                   },
+                    "name": "simple_csv",
+                    "schema": {},
+                    "url": "http://example.com/simple_csv.txt"}
+               ],
+               "retriever": "True",
+               "retriever_minimum_version": "2.0.dev",
+               "version": "1.0.0",
+               "urls":
+                   {"simple_csv": "http://example.com/simple_csv.txt"}
+               },
+    'expect_out': ['a,b,c', '1,2,3', '4,5,6']
+}
+
+tab_delimeter_csv = {
+    'name': 'tab_delimeter_csv',
+    'raw_data': ['a    b    c',
+                 '1    2    3',
+                 '4    5    6'],
+    'script': {"name": "tab_delimeter_csv",
+               "resources": [
+                   {"dialect": {
+                      "do_not_bulk_insert": "True"
+                   },
+                    "name": "simple_csv",
+                    "schema": {},
+                    "url": "http://example.com/simple_csv.txt"}
+               ],
+               "retriever": "True",
+               "retriever_minimum_version": "2.0.dev",
+               "version": "1.0.0",
+               "urls":
+                   {"simple_csv": "http://example.com/simple_csv.txt"}
+               },
+    'expect_out': ['a   b   c', '1   2   3', '4   5   6']
```

Comment:
` 'expect_out': ['a,b,c', '1,2,3', '4,5,6']` this should be the expected results.
So the retriever will install the dataset and when it comes to testing, it exports the installed data into CSV. 

In this case, we take in tab data and we bring out CSV